src/main/java/ch/uzh/ifi/hase/soprafs22/rest/dto/HealthDTO.java

```diff
@@ -0,0 +1,22 @@
+package ch.uzh.ifi.hase.soprafs22.rest.dto;
+
+public class HealthDTO {
+    private PositionDTO defenderPosition;
+    private long health;
```

Comment:
why the upcasting to a long?
usually our health is an int, that should be enough in my oppinion